xml/docu_styleguide.taglist.xml

```diff
@@ -0,0 +1,1175 @@
+       </para>
+      </entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+indexterm
+ primary
+  secondary
+       </screen>
+      </entry>
+      <entry>
+      <para>Index marker</para>
+      <para>
+       legacy element, do not use for new documentation
+      </para>
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+inlinemediaobject
+       </screen>
+      </entry>
+      <entry>
+       An inline media object (video, audio, image...)
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+keycap
+       </screen>
+      </entry>
+      <entry>
+       <para>The text printed on a key on a keyboard</para>
+       <para>
+        For function keys always use with function, for example: 
+       &lt;keycap function="control">&lt;/keycap>
+       </para>
+       </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+keycombo
+ keycap
+       </screen>
+      </entry>
+      <entry>
+       A combination of input actions on a keyboard
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+link
+       </screen>
+      </entry>
+      <entry>
+       A hypertext link
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+literal
+       </screen>
+      </entry>
+      <entry>
+       To mark up data taken literally from a computer system
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+menuchoice
+       </screen>
+      </entry>
+      <entry>
+       A selection or series of selections from a menu
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+option
+       </screen>
+      </entry>
+      <entry>
+       An option for a software command
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+package
+       </screen>
+      </entry>
+      <entry>
+       Name of a software or application package
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+parameter
+       </screen>
+      </entry>
+      <entry>
+       A value or a symbolic reference to a value
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+phrase
+       </screen>
+      </entry>
+      <entry>
+       A span of text
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+productname
+       </screen>
+      </entry>
+      <entry>
+       <para>
+       The formal name of a product
+       When referring to an application, add a
+       &lt;phrase role="productname"/> element around it, for example:
+       </para>
+       <screen>
+&lt;phrase role="productname">
+LibreOffice&lt;/phrase> is an
+office suite.
```

Comment:
I would remove the linebreaks and try to use one line. If that's not possible, add a linebreak after `</phrase>`:

```suggestion
&lt;phrase role="productname">LibreOffice&lt;/phrase> is an office suite.
```

Reply:
I'll check how it will look in the final PDF